CartApi/Infrastructure/ActionResults/InternalServerErrorObjectResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ShoesOnContainers.Services.CartApi.Infrastructure.ActionResults
```

Comment:
```suggestion
namespace EventBriteAssignment.Services.CartApi.Infrastructure.ActionResults
```
change namespace name, should match with Rebka's namespace name change too